### ISSUE TITLE
React Tiptap Editor

### ISSUE THREAD

**Issue by hunghg255** [hunghg255/quill-resize-image#4]
If you guys use React, you can use this component https://rc-tiptap-editor-playground.vercel.app/

**Comment by djscruggs** [comment]
@hunghg255 I saw TipTap editor on Reddit a few days ago and am interested in using it for a project. Would that be possible?

**Comment by hunghg255** [comment]
yes. you can. i just warning about license from founder tiptap because I used to emoji code from version tiptap pro. So I removed package. I will publish again after that you can use it 😅

**Comment by hunghg255** [comment]
you can check from docs https://reactjs-tiptap-editor.vercel.app/. I just publish and push code but set repo private

**Comment by Ishan5hrestha** [comment]
Hi @hunghg255 , do you think there could be any lisence problem in the long run on using this package for production apps?

**Comment by Ishan5hrestha** [comment]
Thank you for quick response!
Also is there any example repo for the complete toolbar seen here?
![image](https://github.com/user-attachments/assets/b9f0e76e-4edc-4f33-92ec-5c6b89f85d3e)
I have to configure each button going throught the documentation..

**Comment by hunghg255** [comment]
check this repo

https://github.com/hunghg255/reactjs-tiptap-editor-demo

**Comment by Ishan5hrestha** [comment]
Thank you man, You are the best!
